lib/Airport_Challenge/airport.rb

```diff
@@ -0,0 +1,32 @@
+
+class Airport
+  def initialize(capacity: 20)
+    @capacity = capacity
+    @planes = []
+  end
+
+  def land(plane)
+    if Weather.stormy?
+      raise "Cannot land plane when is stormy"
+    else
+      @planes << plane
+    end
```

Comment:
You could write this like:

```
raise "Cannot land plane when is stormy" if Weather.stormy?

@planes << plane
```

Also because you have a method called `planes`, you can just do `planes << plane`